PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/mysql/creation/MySQLCreationAdvanced.java

```diff
@@ -0,0 +1,207 @@
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+ * TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package com.microsoft.azure.toolkit.intellij.mysql.creation;
+
+import com.microsoft.azure.management.mysql.v2020_01_01.ServerVersion;
+import com.microsoft.azure.management.resources.ResourceGroup;
+import com.microsoft.azure.management.resources.Subscription;
+import com.microsoft.azure.toolkit.intellij.appservice.resourcegroup.ResourceGroupComboBox;
+import com.microsoft.azure.toolkit.intellij.appservice.subscription.SubscriptionComboBox;
+import com.microsoft.azure.toolkit.intellij.common.AzureFormPanel;
+import com.microsoft.azure.toolkit.intellij.common.AzurePasswordFieldInput;
+import com.microsoft.azure.toolkit.intellij.common.TextDocumentListenerAdapter;
+import com.microsoft.azure.toolkit.intellij.mysql.AdminUsernameTextField;
+import com.microsoft.azure.toolkit.intellij.mysql.ConnectionSecurityPanel;
+import com.microsoft.azure.toolkit.intellij.mysql.MySQLRegionComboBox;
+import com.microsoft.azure.toolkit.intellij.mysql.ServerNameTextField;
+import com.microsoft.azure.toolkit.intellij.mysql.VersionComboBox;
+import com.microsoft.azure.toolkit.lib.common.form.AzureFormInput;
+import com.microsoft.azure.toolkit.lib.mysql.AzureMySQLConfig;
+import lombok.Getter;
+import org.apache.commons.lang3.StringUtils;
+
+import javax.swing.*;
+import javax.swing.event.DocumentListener;
+import java.awt.event.ItemEvent;
+import java.util.Arrays;
+import java.util.List;
+
+public class MySQLCreationAdvanced extends JPanel implements AzureFormPanel<AzureMySQLConfig> {
+
+    private JPanel rootPanel;
+    private ConnectionSecurityPanel security;
+    @Getter
+    private SubscriptionComboBox subscriptionComboBox;
+    @Getter
+    private ResourceGroupComboBox resourceGroupComboBox;
+    @Getter
+    private ServerNameTextField serverNameTextField;
+    @Getter
+    private MySQLRegionComboBox regionComboBox;
+    @Getter
+    private VersionComboBox versionComboBox;
+    @Getter
+    private AdminUsernameTextField adminUsernameTextField;
+    @Getter
+    private JPasswordField passwordField;
+    @Getter
+    private JPasswordField confirmPasswordField;
+
+    private AzurePasswordFieldInput passwordFieldInput;
+    private AzurePasswordFieldInput confirmPasswordFieldInput;
+
+    private AzureMySQLConfig config;
+
+    MySQLCreationAdvanced(AzureMySQLConfig config) {
+        super();
+        this.config = config;
+        $$$setupUI$$$(); // tell IntelliJ to call createUIComponents() here.
+        init();
+        initListeners();
+        setData(config);
+    }
+
+    private void init() {
+        passwordFieldInput = PasswordValidationUtils.generateAzurePasswordFieldInput(this.passwordField, this.adminUsernameTextField);
```

Comment:
it's weird for a `PasswordValidationUtil` to generate a password input, it should not be PasswordValidationUtil's responsibility.